javascript.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  extends: ['./js rules/base', './js rules/style', './js rules/es2018'].map(
```

Comment:
It has to be `'./js rules/es2019'`, right?
```suggestion
  extends: ['./js rules/base', './js rules/style', './js rules/es2019'].map(
```